plugin/src/main/java/com/denizenscript/denizen/scripts/commands/world/TickCommand.java

```diff
@@ -0,0 +1,121 @@
+    //
+    // To change the tick rate, use the 'rate' argument and input the amount using 'amount'. The tick rate must be a number between 1.0 and 10000.0 (inclusive).
+    // To reset the tick rate to the normal value (20.0), use the 'reset' argument.
+    //
+    // To freeze the tick rate, use the 'freeze' argument. To unfreeze, add the 'cancel' argument.
+    // Freezing the tick rate will cause all entities (except for players and entities that a player is riding) to stop ticking.
+    // This means that entity movement and animations will stop, as well as stop things like crop growth, daylight cycle, etc.
+    //
+    // To step the tick rate while the server is frozen for a specific amount of time, use the 'step' argument and input the amount of ticks using 'amount'.
+    // Tick rate stepping is allowing the tick rate to resume for a specific amount of ticks while the server is frozen.
+    // After the amount of specified ticks have passed, the tick rate will refreeze.
+    // If the server is not frozen when trying to use the 'step' argument, nothing will happen.
+    //
+    // To make the tick rate as fast as possible for a specific amount of time, use the 'sprint' argument and input the amount of ticks using 'amount'.
+    // The tick rate will increase as much as possible without overloading the server for the amount of specified ticks.
+    // For example, if you want to sprint 200 ticks, the tick rate will run 200 ticks as fast as possible and then return to the previous tick rate.
+    // To stop stepping or sprinting early, use the 'cancel' argument.
+    //
+    // The tick rate resets to 20.0 on server restart.
+    // For information about tick rate arguments, see <@link url https://minecraft.wiki/w/Commands/tick>
+    //
+    // @Warning Be careful, this command will affect plugins that depend on tick rate and may cause them to break. For example, setting the tick rate to 1 will cause the <@link event tick> event to fire once per second.
+    //
+    // @Usage
+    // Use to set the tick rate to 30 ticks per second.
+    // - tick rate amount:30
+    //
+    // @Usage
+    // Use to stop stepping early if the server is currently stepping.
+    // - tick step cancel
+    //
+    // @Usage
+    // Use to reset tick rate.
+    // - tick reset
+    //
+    // @Usage
+    // Use to freeze the server.
+    // - tick freeze
+    //
+    // @Usage
+    // Use to unfreeze the server.
+    // - tick freeze cancel
+    // -->
+
+    public TickCommand() {
+        setName("tick");
+        setSyntax("tick [rate/step/sprint/freeze/reset] (amount:<#.#>/cancel)");
+        setRequiredArguments(1, 2);
+        isProcedural = false;
+        autoCompile();
+    }
+
+    public enum TickActions { RATE, STEP, SPRINT, FREEZE, RESET }
+
+    public static void autoExecute(@ArgName("action") TickActions action,
+                                   @ArgName("amount") @ArgPrefixed @ArgDefaultNull ElementTag amount,

```

Comment:
You can simplify a lot if you just set this to a `float` (and cast to `(int)` where needed, removing all the type checks in the process)